api/proto/disperser/disperser.proto

```diff
@@ -115,17 +127,39 @@ message BlobStatusReply {
 	BlobInfo info = 2;
 }
 
-// RetrieveBlobRequest contains parameters to retrieve the blob.
-message RetrieveBlobRequest {
+// GetBlobRequest contains parameters to retrieve the blob.
+message GetBlobRequest {
 	bytes batch_header_hash = 1;
 	uint32 blob_index = 2;
 }
 
-// RetrieveBlobReply contains the retrieved blob data
-message RetrieveBlobReply {
+// GetBlobReply contains the retrieved blob data
+message GetBlobReply {
 	bytes data = 1;
 }
 
+// TODO this is identical to the node.proto definition. Should we share this definition?
+// A request to retrieve a specific chunk.
+message GetChunkRequest {
+	// The hash of the blob's ReducedBatchHeader defined on-chain.
+	bytes batch_header_hash = 1;
```

Comment:
As above, should we support both lookup schemes?

Reply:
Would it make sense to support requesting multiple chunks at once?

Reply:
Exactly how this is implemented depends a lot on whether we will eventually stop supporting lookup by blob header hash. Will make this change once I know the answer to this question.

Supporting multiple requests at once makes a lot of sense to me. All also add that in when I make the change to support lookup by both mechanisms.